resources/profiles/FlyingBear/machine/fdm_klipper_common.json

```diff
@@ -16,11 +16,11 @@
   "cooling_tube_length": "5",
   "cooling_tube_retraction": "91.5",
   "default_filament_profile": [
-      "FlyingBear Generic PLA"
+      "FlyingBear PLA"

```

Comment:
You might want to revert this line? 
As discussed in previous sessions.

Reply:
I fixed this code, please check it